boards/arm/nrf9160_pca10090/nrf9160_pca10090_defconfig

```diff
@@ -0,0 +1,23 @@
+CONFIG_ARM=y
+CONFIG_SOC_FAMILY_NRF=y
+CONFIG_SOC_SERIES_NRF91X=y
+CONFIG_SOC_NRF9160_SICA=y
+CONFIG_BOARD_NRF9160_PCA10090=y
+
+# Enable MPU
+CONFIG_ARM_MPU=y
+
+# Enable TrustZone-M
+CONFIG_ARM_TRUSTZONE_M=y
+
+# Enable UARTE
+CONFIG_SERIAL=y
+CONFIG_UART_NRFX=y

```

Comment:
This is redundant (the option is enabled by default).

Reply:
Thanks, done. And for the "ns" fail, too.